backend/routes/job.js

```diff
@@ -240,35 +225,23 @@
 		Job.addJob(req.params.account, newJob).then(() => {
 			responseCodes.respond(utils.APIInfo(req), req, res, next, responseCodes.OK, newJob);
 		}).catch(err => {
-
 			responseCodes.respond(utils.APIInfo(req), req, res, next, err, err);
 		});
-
 	}
 
 	function updateJob(req, res, next) {
-
-		if(!req.body._id) {
-			return Promise.reject(responseCodes.JOB_ID_INVALID);
-		}
-		Job.findByJob(req.params.account, req.body._id).then(job => {
-			return job.updateJob(req.body).then(() => {
-				responseCodes.respond(utils.APIInfo(req), req, res, next, responseCodes.OK, {});
-			});
+		// FIXME: should consider using req.params.jobId instead of body._id
```

Comment:
Probably irrelevant if we're defining the schema to require the job schema to be in the body.
But maybe we should check that params.jobId === body._id, otherwise throw invalid arugments